src/app/config/config.model.ts

```diff
@@ -78,6 +80,16 @@ interface TPLinkSmartPlugPlugin extends Plugin {
   smartPlugIP: string;
 }
 
+interface TasmotaPlugin extends Plugin {
+  ip: string;
+  idx: number;
```

Comment:
Is `idx` index? Might be beneficial to have `index` as the attribute name instead to prevent confusions.

Reply:
idx changed to index throughout